src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -213,6 +213,16 @@ public Size ClientSize
             }
         }
 
+        public Size FrameSize
+        {
+            get
+            {
+                DwmGetWindowAttribute(_hwnd, (int)DwmWindowAttribute.DWMWA_EXTENDED_FRAME_BOUNDS, out var rect, Marshal.SizeOf(typeof(RECT)));
```

Comment:
What happens if DWM composition is disabled?

Reply:
Fixed